sympy/core/mul.py

```diff
@@ -1902,7 +1902,7 @@ def coeff_exp(term, x):
 
         try:
             for t in self.args:
-                coeff, exp = t.leadterm(x, logx=logx)
+                coeff, exp = t.leadterm(x)
```

Comment:
Why should the `logx` parameter not be passed? Is it not required, or giving a wrong result?

Reply:
Yeah it is not required and also would give many wrong result . It went unnoticed as we weren't exactly dealing with the `logx` parameter, but now that the PR introduces the real essence of `logx`, having a `logx = logx` there changes the path/execution of calls involving expressions of type `Mul` where `logx` is not `None`